utils/zerovec/src/hashmap/mod.rs

```diff
@@ -18,7 +18,7 @@ pub use algorithms::*;
 /// use zerovec::ZeroHashMap;
 ///
 /// let kv = vec![(0, "a"), (1, "b"), (2, "c")];
-/// let hashmap: ZeroHashMap<i32, str> = ZeroHashMap::build_from_iter(kv.into_iter());
+/// let hashmap: ZeroHashMap<i32, str> = ZeroHashMap::from_iter(kv.into_iter());
```

Comment:
It's more idiomatic to use `collect` to access `FromIterator`:

```rust
let hashmap = kv.into_iter().collect::<ZeroHashMap<i32, str>>();
```

If you want this to look like a constructor, you could still shorten this:
```rust
let hashmap = ZeroHashMap::<i32, str>::from_iter(kv.into_iter());
```

Also multiple cases below.